src/Symfony/Component/Notifier/Bridge/MicrosoftTeams/Action/ActionCard.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Notifier\Bridge\MicrosoftTeams\Action;
+
+use Symfony\Component\Notifier\Bridge\MicrosoftTeams\Action\Input\InputInterface;
+
+/**
+ * @author Edouard Lescot <edouard.lescot@gmail.com>
+ * @author Oskar Stark <oskarstark@googlemail.com>
+ *
+ * @see https://docs.microsoft.com/en-us/outlook/actionable-messages/message-card-reference#actioncard-action
+ */
+final class ActionCard implements ActionInterface
+{
+    protected $options = ['@type' => 'ActionCard'];
```

Comment:
No need for protected, the class is final

Reply:
Done